[Accessibility] Fixed the color contrast of the tutorial

veler opened the pull request:
Fixed the Orange color contrast of the tutorial by using the Microsoft color.

Related issue : [https://github.com/Microsoft/pxt/issues/2535](https://github.com/Microsoft/pxt/issues/2535)

![untitled](https://user-images.githubusercontent.com/3747805/30400199-f83969c6-988a-11e7-879d-b4fc42cdf22b.png)

msftclas commented:
@veler,
Thanks for your contribution as a Microsoft full-time employee or intern. You do not need to sign a CLA.
_Thanks,
Microsoft Pull Request Bot_

pelikhan commented:
It's very very red now. Isn't that too scary?

abchatra commented:
Can we try a different color?

microsoftsam commented:
This PR description doesn't make sense. You're updating the block color, yet you're showing the color change for the getting started button. The two should be independent.

veler commented:
Hum, in fact, this change affect everything in orange : the block, the buttons (Getting started, Finish) and the orange top bar of the tutorial.
Do you want me to do the change only on the blocks? It would makes sense to me to change it on the buttons too as there is the same color contrast issue than on the block even if the accessibility team don't talk about the buttons explicitly.

microsoftsam commented:
The issue in question here says the "Ok" and the text in orange in the getting started experience, ie the blocks are the issue here. They use an orange color that is non compliant given the text size and the orange color chosen.
You've already fixed the "Ok" button in a separate change, so I think this change should only fix the block text orange to use the MS color in all targets.
We are not able to change the micro:bit orange without Foundation consolation, let's discuss offline if that is necessary.

veler commented:
I changed the color of the non-categorized inline block to use the MS orange color. I closed the PR related to micro:bit [https://github.com/Microsoft/pxt-microbit/pull/539](https://github.com/Microsoft/pxt-microbit/pull/539).
I updated the screenshot.

microsoftsam commented on theme/themes/pxt/globals/site.variables:
```diff
@@ -18,6 +18,7 @@
 @grey:#505050;
 
 @greenTextColor: #107C10;
+@microsoftorange:#D83B01;
```
No need to specify here. use the color directly. or rename to something more descriptive of it's use and place it at the top of tutorial.less

veler commented:
TODO once merged : Do the same PR for Master, or cherry-pick to accessibility-master.

microsoftsam commented on theme/tutorial.less:
```diff
@@ -5,6 +5,8 @@
 /* Reference import */
 @import (reference) "semantic.less";
 
+@inlineblockmicrosoftorange: #D83B01; // The official Microsoft orange color.
```
change to inlineBlockColor..

microsoftsam replied:
No need for comment.
